FreeRTOS-Plus/Demo/FreeRTOS-IoT-Libraries-LTS-Beta2/pkcs11/mqtt_mutal_auth_with_pkcs11/DemoTasks/MutualAuthMQTTExample.c

```diff
@@ -378,15 +378,15 @@ static void prvMQTTDemoTask( void * pvParameters )
         xMQTTStatus = MQTT_ProcessLoop( &xMQTTContext, mqttexamplePROCESS_LOOP_TIMEOUT_MS );
         configASSERT( xMQTTStatus == MQTTSuccess );
 
-        /**************************** Publish and Keep Alive Loop. ******************************/
+        /******************* Publish and Keep Alive Loop. *********************/
         /* Publish messages with QoS1, send and process Keep alive messages. */
         for( ulPublishCount = 0; ulPublishCount < ulMaxPublishCount; ulPublishCount++ )
         {
             LogInfo( ( "Publish to the MQTT topic %s.\r\n", mqttexampleTOPIC ) );
             prvMQTTPublishToTopic( &xMQTTContext );
 
-            /* Process incoming publish echo, since application subscribed to the same
-             * topic, the broker will send publish message back to the application. */
+            /* Process incoming publish echo, since application subscribed to the
+             * same topic, the broker will send publish message back to the application. */
```

Comment:
If we're editing these comments anyway, should we add some articles? E.g. "the broker will send *the* publish message back"

Reply:
Please let's do that in a separate PR.